src/components/GameItem.js

```diff
@@ -4,26 +4,49 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  staticpropTypes = {
```

Comment:
This should be:
```suggestion
  static propTypes = {
```

What you did won't cause errors but also disable checking.